### DISCUSSION TITLE
yo. help me please(

### DISCUSSION THREAD

**Discussion by MMikkey66** [Anime4000/RTL960x#42] [Q&A] [answered]
i bought (GPON SFP ONU Stick с MAC SC this one from ali)
i done the downgrade to 210702 by the webinterface.
but know i understand that i need 220304 for gpon using (PLOAM passwaord and Serial number change) but now i cant do it with webinterface
can someone help me pliz? 
i m a bit newbee xD

**Comment by Anime4000** [comment]
telnet can access?

**Reply by MMikkey66** [reply]
yeah. have unlocked telnet on my windows, but im rly useless in it
by the way, wanted to ask a "step by step" guide for stupid users like me)))
sry and big thanks for you

**Reply by Anime4000** [reply]
having telnet access is nice to have, try this

example, changing firmware partition, first view current active partition:
```
# nv getenv sw_active
sw_active=1
# nv getenv sw_commit
sw_commit=1
```

if show `1` then type this to use other partition:
```
nv setenv sw_active 0
nv setenv sw_commit 0
```

if show `0`, simple change to `1`

then `reboot`

**Reply by MMikkey66** [reply]
![forgithub](https://user-images.githubusercontent.com/107882352/175048686-e231bdf1-301b-439b-8110-79d985d67e0c.jpg)
okey, i can switch between firmwares.
but there is a problem for me - the web interface is the same for both firmwares..
now i cant write a PLOAM pass and serial number for connect my PON provider on any firmware(
Should i do it by the telnet now?

**Reply by MMikkey66** [reply]
wanna interface like this
![123](https://user-images.githubusercontent.com/107882352/175049444-927a3717-1161-4cfd-9153-30272b74daba.jpg)
took it from the alliexpres comments from my stick....

**Reply by MMikkey66** [reply]
![image](https://user-images.githubusercontent.com/107882352/175049873-811588b9-f0ab-4fdf-9a56-6ec19573f5af.png)
or like this...

**Reply by MMikkey66** [reply]
i dont have a big menu on the leftside in my web interface.. why?(
btw 210702 may be good for me, but my webinterface is bad..)

**Reply by Anime4000** [reply]
probably bad firmware stored in the SPI Flash.
it's better do it via telnet for everything

**Reply by MMikkey66** [reply]
it looks impossible for me(
or there is a guide on your github?

**Comment by MMikkey66** [comment, marked as answer]
it looks inpossible or me(
or there is a guide on your github?

**Reply by Anime4000** [reply]
Yes, Put many guide to setup the stick
https://github.com/Anime4000/RTL960x#guide-links-info

**Reply by MMikkey66** [reply]
may be i can try hard reset?![image](https://user-images.githubusercontent.com/107882352/175314322-797fd2f6-b2b1-4e6d-91f1-1130c03bcd57.png)

**Reply by Anime4000** [reply]
sure why not...

make sure backup `MAC_KEY` and `ELAN_MAC_ADDR`

**Reply by Anime4000** [reply]
join Discord for everyone help :)
https://discord.gg/kQJu8dmT (expired in 7 days)

**Reply by MMikkey66** [reply]
ohhh thank you very much!!
i ll join

**Comment by eugenidev** [comment]
Hi I need your help too! I'd stupidly forgotten to backup MAC_KEY :( How can i restore/generate/grab(from Huawei EchoLife HG8245H) it? Maybe there is a way? `echo -n "hsgq1.9aYOURMACADDRESSUPPERCASE" | md5sum` is correct?

**Reply by Anime4000** [reply]
yes, that's correct
